pkgs/development/libraries/libusb1/default.nix

```
@@ -31,12 +33,23 @@ stdenv.mkDerivation rec {
 
   dontDisableStatic = withStatic;
 
-  configureFlags = lib.optional (!enableUdev) "--disable-udev";
+  configureFlags =
+    lib.optional (enableUdev) "--disable-udev"
```

You missed the `!` when rewriting this line

```suggestion
    lib.optional (!enableUdev) "--disable-udev"
```